api/include/can_protocols/brake_can_protocol.h

```diff
@@ -29,13 +36,13 @@
  * @brief Brake command message (CAN frame) ID.
  *
  */
-#define OSCC_BRAKE_COMMAND_CAN_ID (0x60)
+#define OSCC_BRAKE_COMMAND_CAN_ID (0x52)
```

Comment:
Is renumbering these CAN IDs a breaking change? Will this affect current users of OSCC?

Reply:
In a slightly related topic, is there any way for the OSCC modules to indicate what API version they're running so that the compute host can confirm that it's running a compatible version?

Reply:
It's only a breaking change if someone is injecting raw CAN frames instead of using the API. We're going to put out a notice to customers when this gets put into a release that anyone not using the API will need to be aware of the changes.

We've talked about adding some sort of version checking but it hasn't been spec'd out.

Reply:
@austinjmorlan we could make the first CAN ID in a given range be a message dedicated to debug info or something? @adrienthebo what do you think a small sketch of that might look like?

I know @austinjmorlan is working on storing some info/defaults in the device EEPROM, so there are probably a handful of counters/metrics we could store in there about device version, lifecycle, and runtime.

Reply:
On v1.1.1 CAN ID 0x52 is the throttle enable command.